retail-web/src/repositories/localVideoRepository.js

```diff
@@ -17,7 +17,7 @@ export default {
               "28762499-dd36-4b23-96d3-db3eeeaed548",
               "36ba2aec-a64e-4ac5-911d-a36cbbfa83bf"
             ],
-            "thumb_url": "https://d22w93kkp7ubok.cloudfront.net/images/videos/default_stream0_thumb.png"
+            "thumb_url": "https://d202iz0410uej0.cloudfront.net/images/videos/default_stream0_thumb.png"
```

Comment:
Who is maintaining this CloudFront distribution?

Reply:
and the live video `play_back_url` managed by the origin repo
https://github.com/aws-samples/retail-demo-store/blob/198e98d84c4cceb9b04fc888c6352b1fdc1581dc/videos/default_streams/default_streams.json#L3